pandas/core/generic.py

```diff
@@ -3156,6 +3156,13 @@ def copy(self, deep=True):
         data = self._data.copy(deep=deep)
         return self._constructor(data).__finalize__(self)
 
+    __copy__ = copy
```

Comment:
Minor, but a child class that overrides `copy` won't override `__copy__` without a `def __copy__...` here